draft-ietf-rtcweb-jsep.xml

```diff
@@ -1403,6 +1403,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
               <t>An "a=msid-semantic:WMS" line MUST be added, as specified in
               <xref target="I-D.ietf-mmusic-msid"></xref>, Section 4.</t>
+
+              <t>An "a=ice-options" line, with the "trickle" option, as
```

Comment:
I think you need to say "MUST be added" here; the format here is slightly different than the part below.